web/src/pages/agent/form/title-chunker-form/index.tsx

```
@@ -290,19 +292,44 @@ const TitleChunkerForm = ({ node }: INextOperatorForm) => {
+        >
+          <div className="flex flex-col justify-start items-center">
+            <span
+              className="flex self-start"
+              dangerouslySetInnerHTML={{
+                __html:
+                  method === 'hierarchy'
+                    ? t('flow.hierarchyTip')
+                    : method === 'group'
+                      ? t('flow.groupTip')
+                      : '',
+              }}
```

Biome (2.4.10)

[error] Avoid passing both children and the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtmlWithChildren)